pkg/translator/jaeger/jaegerproto_to_traces.go

```diff
@@ -258,22 +258,24 @@ func jTagsToInternalAttributes(tags []model.KeyValue, dest pcommon.Map) {
 	}
 }
 
-func setInternalSpanStatus(attrs pcommon.Map, dest ptrace.Status) {
+func setInternalSpanStatus(attrs pcommon.Map, span ptrace.Span) {
+	dest := span.Status()
 	statusCode := ptrace.StatusCodeUnset
 	statusMessage := ""
 	statusExists := false
 
-	if errorVal, ok := attrs.Get(tracetranslator.TagError); ok {
+	if errorVal, ok := attrs.Get(tracetranslator.TagError); ok && errorVal.Type() == pcommon.ValueTypeBool {
```

Comment:
If we do *not* see an error flag, we should try to calculate one, if it would yield to "true". What I have in mind is:

- if there's an error flag already, skip this logic altogether
- if the status code is >= 500 and there's no status code, set one to true

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#status

Reply:
this is already being done below, that's why we have `statusExists` flag